content/api/utilities/buffer.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Cypress.Buffer
+---
+
+Cypress automatically includes a [Buffer](https://github.com/feross/buffer)
+polyfill for the browser and exposes it as `Cypress.Buffer`.
+
+Use `Cypress.Buffer` to manipulate binary data, similar to the
+[nodejs Buffer class](https://nodejs.org/api/buffer.html). Instances are
+returned by `cy.readFile()` and `cy.fixture()`, and it is used with
+`.selectFile()`.
+
+## Syntax
+
+```javascript
+Cypress.Buffer.method()
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+Cypress.Buffer.method()
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```javascript
+cy.Buffer.method() // Errors, cannot be chained off 'cy'
+```
+
+## Examples
+
+### Image Fixture
```

Comment:
The "Image Fixture" header seems a bit out of place, since it only seems to apply to the first example, but not the second.